nondex-maven-plugin/src/main/java/edu/illinois/nondex/plugin/NonDexSurefireExecution.java

```diff
@@ -91,6 +91,34 @@ protected void setupArgline() {
 
     }
 
+    @Override
+    protected Xpp3Dom applyNonDexConfig(Xpp3Dom configuration) {
+        return addExcludedGroups(super.applyNonDexConfig(configuration));
+    }
+
+    private Xpp3Dom addExcludedGroups(Xpp3Dom configNode) {
```

Comment:
Also, if you want to test with the project that was failing, you can find that project at: nondex-maven-plugin/target/it/comprehensive/ (after running mvn verify -- it will generate there the concrete pom file and all the symbols from the .../src/it/comprehensive should be resolved.
You can there run mvn nondex:nondex with flags and what not and see the error messages.